Mage/src/main/java/mage/watchers/common/CastSpellLastTurnWatcher.java

```diff
@@ -98,20 +90,11 @@ public void reset() {
     }
 
     public int getAmountOfSpellsAllPlayersCastOnCurrentTurn() {
-        int totalAmount = 0;
-        for (Integer amount : amountOfSpellsCastOnCurrentTurn.values()) {
-            totalAmount += amount;
-        }
-        return totalAmount;
+        return amountOfSpellsCastOnCurrentTurn.values().stream().mapToInt(Integer::intValue).sum();
```

Comment:
`amountOfSpellsCastOnCurrentTurn.values().stream().reduce(0, Integer::sum);`

It's not much different but it's how I would do it.